public/employers/meredith.yml

```diff
@@ -0,0 +1,36 @@
+name: Meredith Corporation
+shortName: Meredith
+ticker: MDP
+wiki: Meredith Corporation
+image: meredith.svg
+officialWebsite: https://ir.meredith.com/company-profile/default.aspx
+industries:
+  - media
+location:
+  city: Des Moines
+  state: IA
+  country: us
+  international: false
```

Comment:
I feel like they should be international, considering they have EU Privacy made up. 

https://www.meredith.com/nmg/eu-privacy

Also this: https://www.prnewswire.com/news-releases/meredith-expands-international-reach-in-europe-middle-east-and-asia-147658535.html